torch/_dynamo/convert_frame.py

```diff
@@ -210,6 +214,10 @@ def _fn(*args: _P.args, **kwargs: _P.kwargs) -> _T:
                 return fn(*args, **kwargs)
             finally:
                 cleanup.close()
+                assert (
```

Comment:
I am going to remove and re-add that stack logic in a later PR. Today, there are issues with device context and setting the global default device: we insert calls into the graph today which directly mutate the torch function mode stack. Once this work is completed we can just trace those and manipulate the symbolic stack only. After I do this I can reinsert the stack clear/restore.